internal/gensnippets/main.go

```diff
@@ -77,6 +78,14 @@ func main() {
 			}
 		}
 	}
+
+	if len(dirs) > 0 {
+		cmd := exec.Command("goimports", "-w", ".")
```

Comment:
Do you think we should use `golang.org/x/sys/execabs` instead?

Reply:
Yes. Done. Thanks!